package.json

```diff
@@ -28,7 +28,8 @@
     "test": "jest",
     "test:ci": "npm run test -- --coverage --no-cache",
     "test:watch": "jest --watch",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "postinstall": "patch-package"
```

Comment:
🤞 this is not executed when CirlcCI applies cached packages (that already have the patch applied)